src/sinol_make/commands/run/__init__.py

```diff
@@ -700,6 +705,39 @@ def set_scores(self):
 
         self.possible_score = self.get_possible_score(self.groups)
 
+    def get_valid_input_files(self):
+        """
+        Returns list of input files that have corresponding output file.
+        """
+        output_tests = glob.glob(os.path.join(os.getcwd(), "out", "*.out"))
+        output_tests_ids = [self.extract_test_id(test) for test in output_tests]
+        valid_input_files = []
+        for test in self.tests:
+            if self.extract_test_id(test) in output_tests_ids:
+                valid_input_files.append(test)
+        return valid_input_files
+
+    def validate_tests(self):
```

Comment:
```suggestion
    def validate_existence_of_outputs(self):
```
The current name is misleading (can be related to inwer) and not descriptive of what the function does. I would say that the function can be split further into smaller functions with better names, which would make the code clearer.

Reply:
In fact, I'm not sure whether the name I've proposed is even accurate.

Reply:
I made some changes, but I didn't come up with a better name for this function